components/MainColor/styles.js

```diff
@@ -0,0 +1,17 @@
+import styled from "styled-components";
+
+export const StyledLabel = styled.label`
+  margin: 25px 0px;
```

Comment:
It is common not to set a unit to a value of 0.
```suggestion
  margin: 25px 0;
```